.github/workflows/release.yaml

```diff
@@ -121,7 +121,7 @@ jobs:
         - '4.0'
         - '5.0'
         java:
-        - '8'
+        - '${{ matrix.variant == ''5.0'' && ''11'' || ''8'' }}'
```

Comment:
Is that really valid? I thought the way to go is to have the existing matrix without 5.0 and then do an explicit include with 5.0 + J11